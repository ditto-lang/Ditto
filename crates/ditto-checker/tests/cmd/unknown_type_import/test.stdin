module Test exports (..)

import Dep (Huh)
