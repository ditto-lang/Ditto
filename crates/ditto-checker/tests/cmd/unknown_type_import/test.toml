bin.name = "ditto-checker-testbin-check-module"
args = ["unknown_type_import.ditto"]
fs.sandbox = true
status = "failed"
