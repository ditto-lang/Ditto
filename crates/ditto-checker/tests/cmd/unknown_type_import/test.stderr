
  × unknown type import
   ╭─[unknown_type_import.ditto:2:1]
 2 │ 
 3 │ import Dep (Huh)
   ·             ─┬─
   ·              ╰── this type isn't exposed?
   ╰────

